recipes/jupytext/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jupytext" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 05203c7efc0c7de6947a401538045a42deddbe0b01ed304905f1c9e293d0e742
+
+build:
+  number: 0
+  entry_points:
+    - jupytext = jupytext.cli:jupytext
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```

Comment:
This can probably be a `noarch: python` package, right?

Reply:
I think so (but I am new to conda)